Purchases/Caching/RCDeviceCache.m

```diff
@@ -21,11 +18,16 @@ @interface RCDeviceCache ()
 
 @end
 
-NSString * RCLegacyGeneratedAppUserDefaultsKey = @"com.revenuecat.userdefaults.appUserID";
-NSString * RCAppUserDefaultsKey = @"com.revenuecat.userdefaults.appUserID.new";
-NSString * RCPurchaserInfoAppUserDefaultsKeyBase = @"com.revenuecat.userdefaults.purchaserInfo.";
+
+#define RC_CACHE_KEY_PREFIX @"com.revenuecat.userdefaults"
```

Comment:
apparently this is the recommended way when you have constants that depend on each other, see https://bytes.vokal.io/objc-string-constants/